main.go

```diff
@@ -188,6 +189,18 @@ func main() {
 			if localBundlePath == "" {
 				return fmt.Errorf("path to local bundle is required")
 			}
+			// If the user passes in more than one component for a
+			// local scan, we're dealing with too much ambiguity.
+			// Otherwise, the local scan needs to run for each
+			// component supplied, when it only maps to a single
+			// component. In this case, the user needs to either
+			// use an empty list (no component filtering) or reduce
+			// their filter to a list of one to remove ambiguity
+			// and allow check payload to safely determine the
+			// component to filter against.
+			if len(components) > 1 {
+				return fmt.Errorf("local scans do not support multiple components, use one or none: %s", components)
```

Comment:
This is a slight departure from what @kolyshkin suggested, but depends on how strict we want to be in allowing users to pass in arguments we won't use.

Reply:
Fine with me.